src/main/java/org/gridsuite/loadflow/server/utils/LoadflowResultsUtils.java

```diff
@@ -0,0 +1,70 @@
+package org.gridsuite.loadflow.server.utils;
+
+import com.powsybl.iidm.network.*;
+import com.powsybl.security.BusBreakerViolationLocation;
+import com.powsybl.security.LimitViolation;
+import com.powsybl.security.NodeBreakerViolationLocation;
+import com.powsybl.security.ViolationLocation;
+
+import java.util.*;
+
+import static com.powsybl.security.ViolationLocation.Type.NODE_BREAKER;
+
+public final class LoadflowResultsUtils {
```

Comment:
Why all methods are public 
It's used only by LoadFlowWorkerService

Reply:
this code has been moved to [powsybl-ws-commons](https://github.com/powsybl/powsybl-ws-commons/pull/84) , to avoid duplicating it for security-analysis-server